src/window.rs

```diff
@@ -26,13 +26,13 @@ use config::Config;
 use input::Input;
 
 const COLOR_TABLE: [i16; 8] = [
+	pancurses::COLOR_WHITE,
```

Comment:
Mind adding a comment here explaining that white should be first? That way it doesn't accidentally get placed back in alphabetical order.